.github/workflows/tests.yml

```
@@ -0,0 +1,75 @@
+name: PHPUnit tests
+run-name: ${{ github.actor }}
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  symfony:
+    name: Symfony 5.4 (PHP ${{ matrix.php-versions }})
+    # https://hub.docker.com/_/ubuntu/
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: true
+      matrix:
+        php-versions: ["7.4"]

```

You should change this to `7.3`. Prod is aaactually running 7.3 (i know....)
```suggestion
        php-versions: ["7.3"]

```